Framework/DataHandling/src/LoadIsawDetCal.cpp

```diff
@@ -271,11 +270,10 @@ void LoadIsawDetCal::exec() {
       ComponentScaling detScaling;
       detScaling.scaleX = CM_TO_M * width / det->xsize();
       detScaling.scaleY = CM_TO_M * height / det->ysize();
-      detScaling.compID = det->getComponentID();
+      detScaling.componentName = det->getFullName();
```

Comment:
I didn't know this method. Is its return value compatible with everything we are using it for?